src/test/java/org/jabref/logic/importer/WebFetchersTest.java

```diff
@@ -56,7 +56,7 @@ private Set<Class<?>> getIgnoredInaccessibleClasses() {
                              return Class.forName(classPath);
                          } catch (ClassNotFoundException e) {
                              LOGGER.error(e.getLocalizedMessage());
-                             LOGGER.error("Some of the ignored classes were not found");
+                             LOGGER.error("Some of the ignored classes were not found {}" e);
```

Comment:
You need a comma here for the paramter e

Reply:
Done. For some reason, the linter thinks there's no argument provided to `LOGGER.error()`. Weird, but it's only a warning...

![image](https://user-images.githubusercontent.com/69732601/200659285-5a91ab4f-532c-4b00-9e8c-79e507780ee5.png)